kanidm_book/src/security_hardening.md

```diff
@@ -72,10 +75,11 @@ settings. For example:
     -rw-r--r--  1 william  wheel  0 29 Jul 12:25 test
 
 Notice that even though the file was set to "read only" to william, and no permission to any
-other users, that as "william" I can change the bits to add write permissions back or permissions
+other users, user "william" can change the bits to add write permissions back or permissions
 for other users.
+<!-- gotta watch out for that william dude, he is trouble -->
```

Comment:
Past William is trouble; future William's a real go-getter. He's going to fix everything!

Reply:
> > Overall this book is quite good, it follows a useful, logical order and gives a lot of good examples
> 
> heart Yay! We did good!

Yes you did!